composables/useIcon.ts

```diff
@@ -20,5 +22,23 @@ export const useIcon = () => {
     isDarkMode.value ? '/unlockable-dark.svg' : '/unlockable.svg',
   )
 
-  return { getTokenIconBySymbol, signUpVoucherIcon, unlockableIcon }
+  const getChainIcon = (prefix: Prefix | null): string | null => {
+    switch (prefix) {
+      case 'ahp':
+        return isDarkMode.value ? '/chain/ahp_dark.svg' : '/chain/ahp.svg'
+      case 'ahk':
+        return '/token/kusama_asset_hub.svg'
```

Comment:
ksm does not have dark mode ?

Reply:
done 299768d

<img src="https://github.com/user-attachments/assets/a4e31dae-748a-4306-b3e2-d378ed19d1f9" width="300" />

<img src="https://github.com/user-attachments/assets/517030d2-05b4-4577-974e-3302f2c27111" width="300" />